### PR TITLE
Generate thumbnails command

### PR DESCRIPTION
**Pull request by jakob-o**
Sorry, I f***ed up the rebase in PR #853 here is a new one

### CODE REVIEW

**Comment by vstoykov** [comment]
It's still bad. I think that you will need to create the following:

 1. Create new temporary branch from your current develop.
 2. Hard reset your develop with divio's develop
 3. Cherry pick the commit from your temporary branch
 4. Push your devlop with force in your github repo

**Comment by jakob-o** [comment]
Mhh, I already 
- hard reset to a clean commit (before my first commit)
- force pushed
- merged divio/develop
- created a new commit and pushed
But I will try your suggestion later

**Comment by jakob-o** [comment]
@vstoykov Yay

**Comment by vstoykov** [comment]
@yakky I saw that you where ok with changes but the branch had conflicts. Now they are resolved.

File: filer/management/commands/generate_thumbnails.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
+'''
+Created on May 31, 2016
+
+@author: jakob
+'''
+
```
Comment (yakky) [inline filer/management/commands/generate_thumbnails.py]: Would you remove this?

File: filer/management/commands/generate_thumbnails.py [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+from django.core.management.base import BaseCommand
+from filer.models.imagemodels import Image
+
+
+class Command(BaseCommand):
+
+    def handle(self, *args, **options):
+        pks = Image.objects.all().values_list('id', flat=True)
```
Comment (yakky) [inline filer/management/commands/generate_thumbnails.py]: I read the rationale on the previous PR and I'm ok with this implementation
Would you add a comment on the reason for this syntax?

**Comment by jakob-o** [comment]
@yakky Done

**Pull request merged**